<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>951</width>
      <height>241</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Wachtwoord</string>
     </attribute>
     <widget class="QLabel" name="lb_password_length">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>length</string>
      </property>
     </widget>
     <widget class="QSlider" name="slider_password">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>671</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>256</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="spin_password">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>30</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>571</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Letters and Symols</string>
      </property>
      <widget class="QCheckBox" name="check_symbols6">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>60</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>{ [ ( ) ] }</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="check_symbols5">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>60</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt; * + ! ? =</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="check_numbers">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>30</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>0 - 9</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="check_symbols1">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>30</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string># $ % &amp; @ ^ ` ~</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="check_symbols2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>. , : ; </string>
       </property>
      </widget>
      <widget class="QCheckBox" name="check_az_lower">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>a - z</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="check_symbols4">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>60</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>\  / | _ -</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="check_az_capital">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>A - Z</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="check_symbols3">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>60</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>&quot; '</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Wachtwoordzin</string>
     </attribute>
     <widget class="QSlider" name="slider_passphrase">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>671</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>40</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lb_passphrase_length">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_passphrase_seperator">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_passphrase_seperator">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>.</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>-</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>*</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>_</string>
       </property>
      </item>
     </widget>
     <widget class="QSpinBox" name="spin_passphrase">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>30</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lb_passphrase_text">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_passphrase_text">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Eerste Letter Als Hoofdletter</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>kleine letters</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>HOOFDLETTERS</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="lb_passphrase_lang">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_passphrase_lang">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Nederlands</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>English</string>
       </property>
      </item>
     </widget>
     <widget class="QCheckBox" name="check_passphrase_numbers">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>271</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>End with 4 numbers</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="lb_grade">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>grade</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_generated_password">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>761</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_generate">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>pb_generate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_copy">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>pb_copy</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>761</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="format">
     <string>%p%</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1037</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
